src/dos/program_mousectl.cpp

```diff
@@ -377,8 +377,8 @@ bool MOUSECTL::CmdMap()
 		WriteOut(convert_ansi_markup("[color=cyan]%-4s[reset]   ?").c_str(),
 		         MouseControlAPI::GetInterfaceNameStr(interface_id).c_str());
 
-		uint8_t device_id = 0;
-		if (!mouse_config_api.ProbeForMapping(device_id)) {
+		uint8_t device_idx = 0;
+		if (!mouse_config_api.MapInteractively(interface_id, device_idx)) {
```

Comment:
Kudos also here in the rewording. :+1: 

It's now sufficiently self-documenting that I clearly understand we're asking the user to pick the physical mouse they'd like to map to the DOS device. 

And we'll get back a populated `device_idx`.